test-tool/test_write12_residuals.c

```diff
@@ -25,16 +25,34 @@
 #include "iscsi-private.h"
 #include "scsi-lowlevel.h"
 #include "iscsi-test-cu.h"
-
+#include "test_write_residuals.h"
 
 void
 test_write12_residuals(void)
 {
-        struct scsi_task *task_ret;
-        unsigned char buf[10000];
-        struct iscsi_data data;
-        int ok;
-        unsigned int i;
+        /* testing scenarios */
+        const struct residuals_test_data write12_residuals[] = {
```

Comment:
Same comment here and in the other source files: please change 'const' into 'static const'.